scanapp/tasks.py

```diff
@@ -57,6 +62,42 @@ def scan_code_async(url, scan_id, path, file_name):
 
 
 @app.task
+def scan_code_async_final(url, scan_id):
+    """
+    Create and save a file at `path` present at `URL` using `scan_id` and bare `path`
+    and apply the scan.
+    """
+    logger = logging.getLogger(__name__)
+    logger.info('git repo detected')
+
+    clean_url = ''.join(e for e in url if e.isalnum())
+
+    dir_name = clean_url
+
+    home_path = expanduser("~")
```

Comment:
Saving under a user dir down not make sense. We need a well defined configured place.